pkg/file/pe.go

```diff
@@ -50,9 +50,37 @@ func ExtractPEMetadata(r io.Reader) (*InstallerMetadata, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error parsing PE version resources: %w", err)
 	}
-	return &InstallerMetadata{
+	return applySpecialCases(&InstallerMetadata{
 		Name:    strings.TrimSpace(v["ProductName"]),
 		Version: strings.TrimSpace(v["ProductVersion"]),
 		SHASum:  h.Sum(nil),
-	}, nil
+	}, v), nil
+}
+
+var exeSpecialCases = map[string]func(*InstallerMetadata, map[string]string) *InstallerMetadata{
+	"Notion": func(meta *InstallerMetadata, resources map[string]string) *InstallerMetadata {
```

Comment:
Having the `resources` dictionary here is not strictly necessary, but I added it for easier future special-cases. Worse case it's a very small YAGNI, but I'm happy to delete it too if y'all hate it :D . Just seems like a likely useful thing to have available.